### PR TITLE
update the docs for appConfig

### PR DESCRIPTION
**Pull request by JoeChapman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -161,6 +161,7 @@ HOF-bootstrap accepts the following options so a developer can customise element
 - `fields`: Location of the common fields relative to the root of your project. Defaults to 'fields'. Set `fields` to `false` if not present.
 - `translations`: Location of the common translations relative to the root of your project. Defaults to 'translations'.
 - `baseController`: The base controller for all routes and steps. Defaults to [HOF-controllers.base](https://github.com/UKHomeOfficeForms/hof-controllers/blob/master/lib/base-controller.js).
+- `appConfig`: Allows you to attach a configuration object to each controllers' options argument.
```
Comment (sulthan-ahmed) [inline README.md]: Sorry what does this mean?  Is there an example

Comment (JoeChapman) [inline README.md]: I can add one.

Comment (sulthan-ahmed) [inline README.md]: 👍

**Comment by sulthan-ahmed** [comment]
LGTM

File: README.md [hunk 2]
```diff
@@ -161,7 +161,15 @@ HOF-bootstrap accepts the following options so a developer can customise element
 - `fields`: Location of the common fields relative to the root of your project. Defaults to 'fields'. Set `fields` to `false` if not present.
 - `translations`: Location of the common translations relative to the root of your project. Defaults to 'translations'.
 - `baseController`: The base controller for all routes and steps. Defaults to [HOF-controllers.base](https://github.com/UKHomeOfficeForms/hof-controllers/blob/master/lib/base-controller.js).
-- `appConfig`: Allows you to attach a configuration object to each controllers' options argument.
+- `appConfig`: Allows you to attach a configuration object to each controllers' options argument. Useful if you need to access properties of your applications config settings in other parts of your code base, e.g:
+```
+...
+constructor(options) {
+  this.emailSettings = options.config.emailSettings;
```
Comment (joefitter) [inline README.md]: is this options.config or options.appConfig?

Comment (sulthan-ahmed) [inline README.md]: good spot 👍 @joefitter

Comment (JoeChapman) [inline README.md]: good spot

**Pull request merged**